pyramid_swagger/tween20.py

```diff
@@ -0,0 +1,224 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from collections import namedtuple
+import logging
+
+from bravado_core.request import RequestLike, unmarshal_request
+from pyramid.interfaces import IRoutesMapper
+
+from pyramid_swagger.exceptions import RequestValidationError
+from pyramid_swagger.exceptions import ResponseValidationError
+from pyramid_swagger.model import PathNotMatchedError
+from pyramid_swagger.tween import get_exclude_paths, should_exclude_request, \
+    _get_validation_context
+from pyramid_swagger.tween import validation_error
+
+
+log = logging.getLogger(__name__)
+
+
+class Settings(namedtuple(
+    'Settings',
+    [
+        'spec',
+        'validate_request',
+        'validate_response',
+        'validate_path',
+        'exclude_paths',
+        'exclude_routes',
+    ]
```

Comment:
This looks identical to  the one in `tween.py`, (except for spec vs schema name). Could we use the one in tween.py for now, and rename it to `schema` to `spec` at some point later?